x-pack/metricbeat/module/aws/_meta/config.yml

```diff
@@ -1,18 +1,27 @@
 - module: aws
   period: 300s
   metricsets:
-    - "ec2"
-    - "sqs"
+    - ec2
   access_key_id: '${AWS_ACCESS_KEY_ID:""}'
   secret_access_key: '${AWS_SECRET_ACCESS_KEY:""}'
   session_token: '${AWS_SESSION_TOKEN:""}'
-  default_region: '${AWS_REGION:us-west-1}'
+- module: aws
+  period: 300s
+  metricsets:
+    - sqs
+  access_key_id: '${AWS_ACCESS_KEY_ID:""}'
+  secret_access_key: '${AWS_SECRET_ACCESS_KEY:""}'
+  session_token: '${AWS_SESSION_TOKEN:""}'
+  regions:
```

Comment:
This should default to query from all regions, I suggest to keep `regions` setting here but commented out

Reply:
Good point, I just updated the config.yml with regions commented.